NitroxModel/Server/ServerConfig.cs

```diff
@@ -192,5 +192,7 @@ public ServerSerializerMode SerializerModeEnum
         #endregion
 
         public bool IsHardcore => GameModeEnum == ServerGameMode.HARDCORE;
+
+        public bool IsPasswordRequired => ServerPassword != "";
```

Comment:
minor : string.Empty

Reply:
Imo this isn't usually worth to bring up in review. `""` is just as clear as `string.Empty`. They are also both equivalent in terms of what IL is generated.